extensions/amp-analytics/analytics-vars.md

```diff
@@ -0,0 +1,105 @@
+# <a name="amp-analytics"></a> Variables supported in `amp-analytics`
+
+## Using the `vars`
+
+The format `${var-name}` can be used when a page or platform defined variable needs to be used in a request string. The `amp-analytics` tag will replace the template with the actual value at the time of construction of the analytics request. All the values substituted using this method are first url encoded using `encodeUrlComponent`. For example:
+
+```html
+<amp-analytics>
+<script type="application/json">
+{
+  "requests": {
+    "pageview": "https://example.com/analytics?url=${canonicalUrl}&title=${title}&acct=${account}",
+    ...
+  },
+  "vars": {
+    "account": "ABC123"
+  },
+  "triggers": {
+    ...
+  }
+}
+</script>
+</amp-analytics>
+```
+
+## Supported Vars
+
+**random** provides a random value every time a request is being constructed.
+
+Example value: `0.12345632345`
+
+**canonicalUrl** provides the canonical URL of the current document.
+
+Example value: `http%3A%2F%2Fexample.com%3A8000%2Fanalytics.html`
+
+**canonicalHost** provides the canonical URL's host.
+
+Example value: `example.com`
+
+**canonicalPath** provides the canonical URL's path part.
+
+Example value: `%2Fanalytics.html`
+
+**clientId** provides a per document-source-origin (The origin of the website where you publish your AMP doc) and user identifier. The client id will be the same for the same user if they visit again within one year. If the AMP document is not served through the AMP CDN, the client id will be replaced with a cookie of the name of the cid scope below. If the cookie is not present, an empty string will be returned.
+
+Please see below the required and optional arguments you may pass into `clientId` like a function. (spaces between arguments and values are not allowed)
+
+arguments:
+
```

Comment:
Once you make the variables their own headers, you should be able to better format these lower level items.